### PR TITLE
SPUI-132 > Add emailVerification and Success screens

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: lib/auth/screens/email_verification_success/components/email_is_confirmed_text.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class EmailIsConfirmedText extends StatelessWidget {
+  const EmailIsConfirmedText({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const Text(
+      'Your email address danilkin2ua@gmail.com is confirmed',
```
Comment (VovaMedvediev) [inline lib/auth/screens/email_verification_success/components/email_is_confirmed_text.dart:11]: Why email address is hardcoded?

File: lib/auth/shared/open_my_email_button.dart [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const OpenMyEmailButton({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AuthButtonGrey(
+      text: 'Open My Email',
+      onTap: () {
+        if (Platform.isAndroid) {
```
Comment (VovaMedvediev) [inline lib/auth/shared/open_my_email_button.dart:21]: It is better to extract business logic from UI

File: lib/auth/screens/email_verification_success/components/email_is_confirmed_text.dart (hunk 1 above)
Comment (eli1stark) [reply lib/auth/screens/email_verification_success/components/email_is_confirmed_text.dart:11]: Will be added in the next commits

File: lib/auth/shared/open_my_email_button.dart (hunk 2 above)
Comment (eli1stark) [reply lib/auth/shared/open_my_email_button.dart:21]: Questionable, doesn't hold any business logic, and doesn't serve any purpose outside of the widget's scope.
Maybe in the future when we'll have similar logic will make sense to refactor this into function.

**Pull request merged**